package.json

```diff
@@ -16,7 +16,7 @@
     "esbuild": "npm run esbuild-base -- --sourcemap",
     "esbuild-watch": "npm run esbuild-base -- --sourcemap --watch=forever &",
     "pre:install-extension": "rm sway-vscode-plugin.vsix",
-    "install-extension": "npm run package && code --install-extension sway-vscode-plugin.vsix --force",
+    "install-extension": "npm run package && code --install-extension sway-vscode-plugin-dev.vsix --force",
```

Comment:
I think we should change L24 to `sway-vscode-plugin.vsix` instead of changing this.